pandas/tests/frame/test_arithmetic.py

```diff
@@ -1,12 +1,17 @@
 # -*- coding: utf-8 -*-
+import operator
```

Comment:
aren't you moving things *to* ``pandas/tests/arithmetic`` ?

Reply:
The focus is on tests that can be shared across Index/Series/DataFrame(/EA).  `flex` ops don't satisfy that criterion.

Reply:
In case unclear, the tests added to tests.frame.test_arithmetic were moved from tests.frame.test_operators; so this is a strictly better location.

Reply:
yeah its ok.